src/main/java/billgates/view/gui/BillTable.java

```diff
@@ -0,0 +1,33 @@
+package billgates.view.gui;
+
+import javax.swing.*;
+import java.awt.*;
+
+public class BillTable extends JTable {
+
+    public static final int DEFAULT_HEADER_HEIGHT = 50;
```

Comment:
For this, do we want to keep it at a fixed size? Maybe a fixed size will not work as ideal on larger resolution devices. 
E.g. On 4k resolution, it might be incredibly small. 

Maybe try writing it to a specific percentage of the Window size? 

Also, for the font size, if we decide to make the window resizeable / make the headers adapt to Window size in higher resolutions, we might need to figure out a specific formula to obtain the font size :3

Reply:
@ScottCTD